packages/jsii-ruby-runtime/package.json

```diff
@@ -32,10 +32,10 @@
     "package": "package-ruby"
   },
   "devDependencies": {
-    "jsii-build-tools": "file:../jsii-build-tools",
-    "jsii-calc": "file:../jsii-calc",
-    "jsii-pacmak": "^0.19.0",
-    "jsii-runtime": "^0.19.0",
+    "jsii-build-tools": "*",
```

Comment:
All the `*`s are somewhat scary to me. I understand why it doesn't matter here, but there are cases in which it DOES matter, and how will we ensure that everyone editing these files knows the difference AND doesn't make mistakes?

Reply:
Well I felt people were generally in favor of not incurring so many changes on those, but happy to revert those if that scares you...